### PR TITLE
Added 'pl0t' package

### PR DESCRIPTION
**Pull request by al6x**

### CODE REVIEW

File: packages.json [hunk 1]
```diff
@@ -23156,5 +23156,22 @@
...
+      "plot",
+      "chart",
+      "table",
+      "excel",
+      "spreadsheet",
+      "visualization",
+      "data"
+    ],
+    "description": "Plot and visualize data",
+    "license": "MIT",
```
Comment (dom96) [inline packages.json:23174]: This doesn't look like MIT: https://github.com/al6x/pl0t/blob/main/license

File: packages.json [hunk 2]
```diff
@@ -23156,5 +23156,22 @@
     "description": "A Nim client for Todoist's REST API",
     "license": "Apache-2.0",
     "web": "https://ruivieira.github.io/nim-todoist/index.html"
+  },
+  {
+    "name": "pl0t",
+    "url": "https://github.com/al6x/pl0t?subdir=api/nim",
```
Comment (dom96) [inline packages.json:23162]: since this is a pl0t repo already, why the subdir?

Comment (al6x) [reply packages.json:23162]: It's monorepo, there are couple of different things, the language APIs reside in `/api` folder, so there going to be `/api/nim` and `/api/typescript`.

File: packages.json (hunk 1 above)
Comment (al6x) [reply packages.json]: Thanks, updated.

**Comment by al6x** [comment]
Conflict resolved, please review.

**Comment by Vindaar** [comment]
@dom96 We allow packages with a license "Proprietary" in `nimble`?

**Comment by dom96** [comment]
yes, why wouldn't we?

**Pull request merged**